kernel/core/include/os.ads

```diff
@@ -162,7 +165,7 @@ is
    pragma Export (C, os_mbx_receive, "os_mbx_receive");
 
    procedure os_mbx_send (status  : out os_status_t;
-                          dest_id :     types.int8_t;
+                          dest_id :     os_task_dest_id_t;
```

Comment:
This is called from C or assembly code and I think we should not make any assumption on the value range of the parameter. In particular this value will be set by a user task that is not into the scope of the kernel. So this is uncontrolled variable.

Reply:
The idea is to get first a right specific type and then add compiler clauses to interface with C.
However os_task_dest_id_t is a subtype of types.int8_t so no worry.

Reply:
Concerning user input value, I agree a control have to be done either in calling code or called code. In the latter, a proposal is: type parameter still remain types.int8_t with a inside test range of os_task_dest_id_t?